docs/root/configuration/http_filters/dynamic_forward_proxy_filter.rst

```diff
@@ -5,12 +5,12 @@ Dynamic forward proxy
 
 .. attention::
 
-  HTTP dynamic forward proxy support should be considered alpha and not production ready. Stats
-  as well as circuit breakers are missing and will be added soon.
+  HTTP dynamic forward proxy support should be considered alpha and not production ready. Circuit
```

Comment:
Aren't stats still missing from cluster/filter?

Reply:
What other stats do you want to see? We have all the existing cluster stats and HTTP stats.

Reply:
(Note that I'm implementing circuit breakers in the following PR which will also expose pending stats)